### DISCUSSION TITLE
Chrome flags

### DISCUSSION THREAD

**Discussion by gomes-web** [3kh0/ext-remover#1091] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
Are there any flags that can be useful, like getting an unblocked browser or disabling extension.

**Comment by Inglan2** [comment]
prob not

**Comment by S-PScripts** [comment]
not really but there is the flag to stop chromeos automatic updates (removed in newer updates i believe?)

Instructions:
1. Go to chrome://flags#show-metered-toggle.
2. Enable it and click restart. For me, it was already set to default (on).
3. Go to settings.
4. Click Wi-Fi.
5. Click the wifi you want to stop updates from happening.
If it's not there then click Known networks - it'll be there.
6. Click Advanced.
7. Enable Metered Network.
https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/CAUB%20Flags
